keras_nlp/models/bert/bert_masked_lm_test.py

```diff
@@ -75,11 +76,13 @@ def test_classifier_predict(self):
         self.masked_lm.predict(self.raw_batch)
         self.masked_lm.preprocessor = None
         self.masked_lm.predict(self.preprocessed_batch[0])
+        self.masked_lm.preprocessor = self.preprocessor
```

Comment:
This part feels not insignificant, we are ruining the hermeticity of these tests for a 25% savings in time here.

Honestly I am not sure that is worth it. I could see this leading to lots of problems.